packages/itwin/tree-widget/package.json

```diff
@@ -61,47 +61,56 @@
     "@itwin/core-markup": "^4.0.0",
     "@itwin/core-orbitgt": "^4.0.0",
     "@itwin/core-quantity": "^4.0.0",
-    "@itwin/core-react": "^4.0.0",
+    "@itwin/core-react": "^4.1.0",
     "@itwin/core-telemetry": "^4.0.0",
     "@itwin/ecschema-metadata": "^4.0.0",
-    "@itwin/eslint-plugin": "^3.7.0",
-    "@itwin/imodel-components-react": "^4.0.0",
+    "@itwin/eslint-plugin": "^4.0.0-dev.38",
+    "@itwin/imodel-components-react": "^4.1.0",
     "@itwin/presentation-backend": "^4.0.0",
     "@itwin/presentation-common": "^4.0.0",
     "@itwin/presentation-components": "^4.0.0",
     "@itwin/presentation-frontend": "^4.0.0",
     "@itwin/presentation-testing": "^4.0.0",
     "@itwin/webgl-compatibility": "^4.0.0",
+    "@rushstack/eslint-patch": "1.2.0",
+    "@testing-library/dom": "^8.12.0",
     "@testing-library/react": "^12.0.0",
     "@testing-library/react-hooks": "^7.0.2",
-    "@types/chai": "4.3.1",
+    "@testing-library/user-event": "14.4.3",
+    "@types/chai": "4.3.5",
     "@types/chai-jest-snapshot": "^1.3.4",
+    "@types/chai-subset": "1.3.3",
     "@types/deep-equal": "^1.0.1",
     "@types/i18next": "^8.4.2",
-    "@types/jsdom-global": "3.0.2",
-    "@types/mocha": "^8.2.2",
+    "@types/jsdom-global": "^3.0.4",
+    "@types/mocha": "^10.0.1",
     "@types/react": "^17.0.19",
     "@types/react-dom": "^17.0.9",
-    "@types/react-redux": "^7.0.1",
-    "@types/sinon": "^9.0.0",
-    "@types/sinon-chai": "^3.2.0",
-    "@typescript-eslint/eslint-plugin": "^5.3.1",
-    "@typescript-eslint/parser": "^5.0.0",
-    "chai": "^4.2.0",
+    "@types/sinon": "^10.0.15",
+    "@types/sinon-chai": "^3.2.9",
+    "@typescript-eslint/eslint-plugin": "^5.59.8",
+    "@typescript-eslint/parser": "^5.59.8",
+    "chai": "^4.3.7",
     "chai-jest-snapshot": "^2.0.0",
+    "chai-subset": "1.6.0",
     "cpx2": "^3.0.0",
     "deep-equal": "^1.0.0",
-    "eslint": "^7.11.0",
-    "jsdom": "^19.0.0",
+    "eslint": "^8.41.0",
+    "eslint-plugin-unused-imports": "^2.0.0",
+    "eslint-plugin-react": "^7.32.2",
+    "jsdom": "^22.1.0",
     "jsdom-global": "3.0.2",
-    "mocha": "^10.0.0",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "mocha": "^10.2.0",
+    "nyc": "15.1.0",
+    "react": "^17.0.0",
+    "react-dom": "^17.0.0",
+    "react-redux": "^7.2.2",
+    "redux": "^4.1.0",
     "rimraf": "^3.0.2",
-    "sinon": "^9.0.2",
-    "sinon-chai": "^3.2.0",
+    "sinon": "^15.1.0",
+    "sinon-chai": "^3.7.0",
     "typemoq": "^2.1.0",
-    "typescript": "~4.4.0",
+    "typescript": "~5.0.0",
     "xmlhttprequest": "^1.8.0"
   },
   "peerDependencies": {
```

Comment:
did you want to change the appui peers to ^4.1.0 ?

Reply:
No, appui peerDependencies should stay at ^4.0.0.